README.md

```diff
@@ -44,6 +46,16 @@ Just use your package managers as you usually would. Run `yarn install` in Yarn
 
 - **If the local project isn't configured for any package manager**, Corepack will assume that you know what you're doing, and will use whatever package manager version has been pinned as "known good release". Check the relevant section for more details.
 
+### When Authoring Packages
+
+Set your package's manager with the `packageManager` field in `package.json`:
+
+```json
+{
+  "packageManager": "yarn@3.1.1",
```

Comment:
We should probably use an up-to-date version (although it will forever drift away to being outdated anyway, so it's not blocking), and use a SHA to validate the download:

```suggestion
  "packageManager": "yarn@3.2.3+sha224.953c8233f7a92884eee2de69a1b92d1f2ec1655e66d08071ba9a02fa"
```